src/lib/__tests__/index.ts

```diff
@@ -108,3 +108,10 @@ test('setExpectations with a different baseUrl for the stub server', () => {
     body: JSON.stringify(mockStub)
   });
 });
+
+test('setSpecmaticEnvVariable updates the environment variable value in the specmatic.json file', () => {
+  const name = "key"
+  const updatevalue = "updated-value"
+  const fileName = "specmatic.json"
+  expect(setSpecmaticEnvVariable(name, updatevalue, fileName))
+});
```

Comment:
A couple of more tests I can think of, what if the key is not present in specmatic.json, how does our method handle that

Reply:
What if specmatic.json doesnt exist, does out method just break, or does it handle an exception gracefully, we can write a test for that as well

Reply:
Similary what happens when file doesnt contain environments key and environments doesn't contain variables key, will our method fail gracefully?
This is how tests can drive better code design

Reply:
Added try-catch to handle the exception for missing file and if the object is not present inside the specmatic.json